tests/ui/pages/desktop/regions/sorter.py

```diff
@@ -0,0 +1,21 @@
+from selenium.webdriver.common.by import By
+
+from pages.desktop.base import Base
+
+
+class Sorter(Base):
+    """Helper class for sorting."""
+    _root_locator = (By.ID, 'sorter')
+    _sort_by_type_locator = (By.CSS_SELECTOR, 'ul > li')
+    _updating_locator = (By.CSS_SELECTOR, '.updating')
+
+    def sort_by(self, sort):
+        """Clicks the sort by type requested."""
+        elements = self.find_elements(*self._sort_by_type_locator)
```

Comment:
Yes I originally wanted to check the sort for items within More. Actions Chains are still weird on Travis though.